modules/turnos/routes/turno.ts

```diff
@@ -163,7 +163,10 @@ router.patch('/turno/:idTurno/bloque/:idBloque/agenda/:idAgenda/', async (req, r
     const continues = ValidateDarTurno.checkTurno(req.body);
     const pacienteTurno = req.body.paciente;
     if (continues.valid) {
-        let agendaRes = await getAgenda(req.body.idAgenda);
+        let agendaRes: any = await getAgenda(req.body.idAgenda);
+        if (agendaRes.estado === 'pausada' || agendaRes.estado === 'suspendida') {
```

Comment:
No sería más conveniente directamente validar si el estado es distinto de 'publicada' (que es el filtro de estado que utiliza originalmente para buscar agendas disponibles)
 ```suggestion
        if (agendaRes.estado !== 'publicada') {
```

Reply:
El PATCH es para toda la aplicación, no se podrían dar turno a agendas disponibles por ejemplos. (con el chequeo a publicada) 
Me queda confirmar si se pueden dar turnos en agendas pendientes de codificar.